config/external_db_map/default.txt

```diff
@@ -32,9 +32,9 @@ DRscDB	DRscDB	gene	gene
 
 EntrezGene	EntrezGene	gene
 
-flybase_annotation_id	flybase_annotation_id	gene
-FlyBase	flybase_annotation_id	gene
-FlyBase_Annotation_IDs	flybase_annotation_id	gene
+flybase_annotation_id	flybase_annot_id	gene
+FlyBase	flybase_annot_id	gene
```

Comment:
Could you add `flybase_annot_id` to `flybase_annot_id` rule as well.
Same for other cases.

Reply:
Of course :facepalm: Cheers!